src/battle_script_commands.c

```diff
@@ -8603,13 +8605,15 @@ static void HandleScriptMegaPrimal(u32 caseId, u32 battlerId, bool32 isMega)
     // Change species.
     if (caseId == 0)
     {
-        if (isMega)
+        if (type == 0)
```

Comment:
Can you add some constant labels for the `type` argument?

That'd make the code more readable :eyes:

Reply:
Oh yeah, that was something I was going to get to but I guess I forgot